js/data.js

```diff
@@ -1,4 +1,7 @@
 var preferences_template = {
+	"showUpdateNotifications":{
+		"default_value": false
```

Comment:
I think they should probably be enabled by default.

Reply:
@AndrewRayCode can you change the default setting value to true? Looks good otherwise.

Reply:
Also, could you move this new preference to the end of the preferences_template object?